bootstrap.sh

```diff
@@ -392,15 +396,18 @@ ensure_docker_server() {
 install_brewfile() {
   if [ -d "$1" ]; then
     log "Installing from sentry Brewfile"
-    cd "$1" && brew bundle -q
-    SENTRY_NO_VENV_CHECK=1 ./scripts/do.sh init-docker
+    # This is useful when trying to run the script on a non-clean machine
+    (cd "$1" && brew bundle) || log "Something failed during brew bundle but let's try to continue"
```

Comment:
Making `brew bundle` a bit more noisy.

Not aborting if it fails is useful when doing development. For instance, it can sometimes fail if I don't first run `./uninstall.sh` and there's `/Applications/Docker.app` left behind.